ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/tasks/ImportBlocksTask.java

```diff
@@ -96,6 +97,11 @@ protected void executeTaskWithPeer(final EthPeer peer) throws PeerNotConnected {
             });
   }
 
+  @Override
+  protected Optional<EthPeer> findSuitablePeer() {
```

Comment:
Wonder if its worth it to require subclasses to implement this...

Reply:
Interesting.  Will take a look at that and follow up with a PR if it works out.